### PR TITLE
[EP-902] Settings UI configuration mapping to JudoSdk configuration

### PR DESCRIPTION
**Pull request by AndreiGalkin**

### CODE REVIEW

File: sample-app/screens/Home/Home.tsx [hunk 1]
```diff
@@ -12,11 +16,97 @@ import {
...
+    this.getData()
+  }
+
+  async getData() {
+    this.setState({ spinner: true })
+    try {
+      const value = await AsyncStorage.getItem(storageKey)
+      if (value !== null) {
+        const settings = JSON.parse(value)
+        console.log("settings " + JSON.stringify(settings)) //TODO REMOVE
```
Comment (mpetrenco) [inline sample-app/screens/Home/Home.tsx:50]: Forgot to remove this one

File: sample-app/screens/Home/Home.tsx [hunk 2]
```diff
@@ -12,11 +16,97 @@ import {
...
+  };
+
+  componentDidMount() {
+    store.subscribe(() => {
+      this.getData()
+    })
+    this.getData()
+  }
+
+  async getData() {
```
Comment (mpetrenco) [inline sample-app/screens/Home/Home.tsx:44]: Is it possible to move the parsing logic into a separate file?

File: sample-app/screens/Home/Home.tsx [hunk 3]
```diff
@@ -12,11 +16,97 @@ import {
...
+          token: token,
+          isSandbox: isSandboxed
+        }, () => {
+          this.setState({ spinner: false })
+        })
+      } else {
+        this.setState({ spinner: false })
+      }
+    } catch(e) {
+      console.log("getData() error " + e)
```
Comment (mpetrenco) [inline sample-app/screens/Home/Home.tsx:83]: Might remove this too

File: sample-app/screens/Home/Home.tsx (hunk 1 above)
Comment (AndreiGalkin) [reply sample-app/screens/Home/Home.tsx:50]: 👍

**Pull request merged**